tests/integration/loopback_test.yaml

```diff
@@ -0,0 +1,34 @@
+description: Delete the loopback interface
+interfaces:
+- name: lo
+  type: loopback
+  state: absent
+description: Set the address to 192.0.2.251/24 and 2001:db8:1::1/64 for loopback device
+interfaces:
+- name: lo
+  type: loopback
+  state: up
+  ipv4:
+    address:
+    - ip: 192.0.2.251
+      prefix-length: 24
+  ipv6:
+    address:
+    - ip: 2001:db8:1::1
+      prefix-length: 64
+description: Delete the loopback interface
```

Comment:
Revert the loopback interface to default.

Reply:
We never allows user to remove loopback interface, it will break the whole system.

Reply:
I updated it, thanks.